docs/sql-ref-syntax-qry-select-aggregate.md

```diff
@@ -0,0 +1,137 @@
+('George Vanauf','Sales',10563),
+('Loui Bondur','SCM',10449),
+('Gerard Hernandez','SCM',6949),
+('Pamela Castillo','SCM',11303),
+('Larry Bott','SCM',11798),
+('Barry Jones','SCM',10586)
+AS basic_pays(employee_name, department, salary);
+
+SELECT * FROM basic_pays;
++-----------------+----------+------+
+|    employee_name|department|salary|
++-----------------+----------+------+
+|      Anthony Bow|Accounting|	6627|
+|      Barry Jones|       SCM| 10586|
+|     Diane Murphy|Accounting|	8435|
+|   Foon Yue Tseng|     Sales|	6660|
+|    George Vanauf|     Sales| 10563|
+|    Gerard Bondur|Accounting| 11472|
+| Gerard Hernandez|       SCM|	6949|
+|    Jeff Firrelli|Accounting|	8992|
+|   Julie Firrelli|     Sales|	9181|
+|       Larry Bott|       SCM| 11798|
+|  Leslie Jennings|        IT|	8113|
+|  Leslie Thompson|        IT|	5186|
+|      Loui Bondur|       SCM| 10449|
+|   Mary Patterson|Accounting|	9998|
+|  Pamela Castillo|       SCM| 11303|
+|  Steve Patterson|     Sales|	9441|
+|William Patterson|Accounting|	8870|
++-----------------+----------+------+
+
+SELECT
+    employee_name,
+    department,
+    salary,
+    percentile_cont(0.25) WITHIN GROUP (ORDER BY salary) OVER w AS p1,
+    percentile_disc(0.25) WITHIN GROUP (ORDER BY salary) OVER w AS p2,
+    percentile_cont(0.25) WITHIN GROUP (ORDER BY salary DESC) OVER w AS p3,
+    percentile_disc(0.25) WITHIN GROUP (ORDER BY salary DESC) OVER w AS p4
+FROM basic_pays
+WINDOW w AS (PARTITION BY department)
+ORDER BY salary;
++-----------------+----------+------+-------+-------+-------+-------+
+|    employee_name|department|salary|     p1|     p2|     p3|     p4|
++-----------------+----------+------+-------+-------+-------+-------+
+|  Leslie Thompson|        IT|  5186|5917.75| 5186.0|7381.25| 8113.0|
+|      Anthony Bow|Accounting|  6627|8543.75| 8435.0| 9746.5| 9998.0|
+|   Foon Yue Tseng|     Sales|	6660|8550.75| 6660.0| 9721.5|10563.0|
+| Gerard Hernandez|       SCM|	6949|10449.0|10449.0|11303.0|11303.0|
+|  Leslie Jennings|        IT|	8113|5917.75| 5186.0|7381.25| 8113.0|
+|     Diane Murphy|Accounting|	8435|8543.75| 8435.0| 9746.5| 9998.0|
+|William Patterson|Accounting|	8870|8543.75| 8435.0| 9746.5| 9998.0|
+|    Jeff Firrelli|Accounting|	8992|8543.75| 8435.0| 9746.5| 9998.0|
+|   Julie Firrelli|     Sales|	9181|8550.75| 6660.0| 9721.5|10563.0|
+|  Steve Patterson|     Sales|	9441|8550.75| 6660.0| 9721.5|10563.0|
+|   Mary Patterson|Accounting|	9998|8543.75| 8435.0| 9746.5| 9998.0|
+|      Loui Bondur|       SCM| 10449|10449.0|10449.0|11303.0|11303.0|
+|    George Vanauf|     Sales| 10563|8550.75| 6660.0| 9721.5|10563.0|
+|      Barry Jones|       SCM| 10586|10449.0|10449.0|11303.0|11303.0|
+|  Pamela Castillo|       SCM| 11303|10449.0|10449.0|11303.0|11303.0|
+|    Gerard Bondur|Accounting| 11472|8543.75| 8435.0| 9746.5| 9998.0|
+|       Larry Bott|    	  SCM| 11798|10449.0|10449.0|11303.0|11303.0|
++-----------------+----------+------+-------+-------+-------+-------+
+```
+
```

Comment:
we should document the `WHERE` syntax in aggregate functions as well.

Reply:
OK